src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/TdsEnums.cs

```diff
@@ -317,20 +334,26 @@ public enum ActiveDirectoryWorkflow : byte
             0x72xx0002 -> 2005 RTM
         */
 
-
-        // 2000 SP1 and beyond versioning scheme:
+        // Pre 2000 SP1 versioning scheme:
+        public const int SQL70OR2000_MAJOR = 0x07;     // The high byte (b3) is not sufficient to distinguish
+        public const int SQL70_INCREMENT = 0x00;     // 7.0 and 2000
+        public const int SQL2000_INCREMENT = 0x01;     // So we need to look at the high-mid byte (b2) as well
+        public const int DEFAULT_MINOR = 0x0000;
 
         // Majors:
+        public const int SQL2000SP1_MAJOR = 0x71;     // For 2000 SP1 and later the versioning schema changed and
         public const int SQL2005_MAJOR = 0x72;     // the high-byte is sufficient to distinguish later versions
         public const int SQL2008_MAJOR = 0x73;
-        public const int SQl2012_MAJOR = 0x74;
+        public const int SQL2012_MAJOR = 0x74;
```

Comment:
This change wasn't made in the original PR. I thought it would be ok to change it since these fields aren't included in the public refs.

Reply:
That's my fault but as problems go it's a small one and I agree that it should have no visible impact.